src/lib/components/templates/Chat.svelte

```diff
@@ -1,14 +1,23 @@
 <script>
 	import Header from './Header.svelte'
 	import type { ChatMessage } from '$types'
-	import { SpokenText, Help, Back } from '$atoms'
+	import { SpokenText, Help, Back, MessageCloud, AudioPlayer } from '$atoms'
 	import { AudioRecorder } from '$organisms'
 	import { client } from '$config/supabase'
+	import { resolveLetter } from '$db/letter'
+	import { postResolveStatusMessage } from '$db/messageHandler'
 
 	export let messages: ChatMessage[]
 	export let userRole: string
+	export let letterId: string
+
+	let form: HTMLFormElement
+	let radioVal: string = ''
+	let submitted: boolean = false
 
 	const userId = client.auth.session().user.id
+
+	$: isUser = userRole === 'user' ? true : false
```

Comment:
`isUser = userRole === 'user' ? true : false` does exactly the same as `isUser = userRole === 'user'`, so you might as well remove the ternary operator.